pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoader.java

```diff
@@ -727,18 +725,18 @@ public String answerCompositeQuestion(Handle handle,
                                           String instanceGuid,
                                           List<String> nestedGuids, List<Integer> compositeAnswerOrders, AnswerDao answerDao) {
 
-        Answer answer = new CompositeAnswer(null, pepperQuestionStableId, null);
-        String guid = answerDao.createAnswer(handle, answer, participantGuid, instanceGuid);
+        Answer parentAnswer = answerDao.createAnswer(participantGuid, instanceGuid,
+                new CompositeAnswer(null, pepperQuestionStableId, null));
         List<Long> childrenAnswerIds = new ArrayList<>();
 
+        var jdbiCompositeAnswer = handle.attach(JdbiCompositeAnswer.class);
         if (CollectionUtils.isNotEmpty(nestedGuids)) {
             for (String childGuid : nestedGuids) {
-                childrenAnswerIds.add(answerDao.getAnswerIdByGuids(handle, instanceGuid, childGuid));
+                childrenAnswerIds.add(answerDao.getAnswerSql().findDtoByGuid(childGuid).get().getId());
```

Comment:
I think `DataLoader` and `StudyDataLoader` are like the only places that uses the answer guids extensively. Might be nice to switch to passing around answer ids or `Answer` objects to get rid of this extra query. CC @ssettipalli